docs/MapColonies/Raster/services/PolygonParts/README.md

```diff
@@ -469,13 +478,14 @@ Now that we hold the structure of the `orthophoto_mosaic_base_ORTHOPHOTO_MOSAIC_
       },
       "geometry_name": "geometry",
       "properties": {
-        "partId": 2,
-        "recordId": "a1b6dd1c-c77c-42e6-a00a-306b67bcdebf",
+        "id": "f7885a03-b669-40cb-8ba7-9410fb644011",
+        "partId": "6e870b46-5b7f-4b81-91f6-f22fc39ab60b",
+        "catalogId": "a1b6dd1c-c77c-42e6-a00a-306b67bcdebf",
         "productId": "ORTHOPHOTO_MOSAIC_BASE",
-        "producType": "OrthophotoBest",
-        "id": "MAS_6_ORT_247568-26.0",
-        "name": "O_south_w84geo_Tiff_10cm",
-        "updatedInVersion": "2",
+        "productType": "OrthophotoBest",
+        "sourceId": "MAS_6_ORT_247568-26.0",
+        "sourceName": "O_south_w84geo_Tiff_10cm",
+        "productVersion": "2.0",
```

Comment:
Was this intentional? Do all of the versions need to be fixed to this format?

Reply:
I think productVersion must be X.X (with up to 2 numbers after the dot) 
but in the new ingestion trigger we set the version to 1.0 and increase it by 1 in update
i will change all the versions to have a .0

Reply:
the point is that they will know that version is a string

Reply:
Great, thank you!